package.json

```diff
@@ -27,18 +27,18 @@
     "@airtap/browserify-istanbul": "^4.0.0",
     "airtap-default": "^1.0.0",
     "airtap-multi": "^1.0.0",
-    "browserify": "^16.5.2",
+    "browserify": "^17.0.0",
     "bruce-millis-option": "^1.0.0",
-    "compression": "^1.7.1",
-    "debug": "^4.1.0",
-    "engine.io": "^3.4.2",
-    "engine.io-client": "3.3.2",
-    "express": "^4.17.0",
+    "compression": "^1.7.4",
+    "debug": "^4.3.1",
+    "engine.io": "^5.1.1",
+    "engine.io-client": "5.1.1",
```

Comment:
I don't have enough information to merge this PR. Please check the following:

1. Is latest `engine.io-client` using ES6+ features? Which is why we currently pin to 3.3.2.
2. What breaking changes were made between `engine.io-client` v3 and v5?
3. Do they support the same browsers (versions)?
4. What breaking changes were made between `engine.io` v3 and v5?